src/Uno.UI/DataBinding/BindingPath.cs

```diff
@@ -194,9 +194,7 @@ public string Path
 			set
 			{
 				if (!_disposed
-					&& _value != null
```

Comment:
This is likely going to have an impact on animations. Let's see what the CI says.

Reply:
Indeed. We need to make the AreDifferent check for a reason that I so far elude me, but I guess we can use the old Value, but I'm not sure if there are specific cases that should actually call the getter instead of using the old value. Let's see what tests fail after the small change I just did.